src/ui/server/controllers/insurance/declarations/anti-bribery/code-of-conduct/index.ts

```diff
@@ -94,7 +95,15 @@ export const post = async (req: Request, res: Response) => {
       return res.redirect(ROUTES.PROBLEM_WITH_SERVICE);
     }
 
-    return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${EXPORTING_WITH_CODE_OF_CONDUCT}`);
+    const answer = req.body[FIELD_ID];
+
+    if (answer === FIELD_VALUES.YES) {
```

Comment:
Ternary operator?

Reply:
I don't think this would be wise, want to check explicitly if it's yes or no